docs/sources/architecture/thing-definition.md

```diff
@@ -154,4 +154,49 @@ The channel group type is defined on the same level as the thing types and chann
 </channel-group-type> 
 ```
 
-When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
\ No newline at end of file
+When a thing will be created for a thing type with channel groups, the channel UID will contain the group ID in the last segment divided by a hash (#). If an Item should be linked to channel within a group, the channel UID would be `binding:multiChannelSwitchActor:myDevice:switchActor1#switch` for the XML example before.
+
+## Properties
+The configuration descriptions explained in full detail [here](../architecture/configuration.md) are used in order to provide a flexible configuration mechanism for bindings. As a binding developer you will often have the demand to add meta data to your thing type that represent no configuration parameter. For this purpose the thing type definition provides the possibility to specify properties, e.g. the vendor or the model ID of the concrete thing:
+
+```xml 
+    <thing-type id="thingTypeID">
+        ...
+        <properties>
+             <property name="vendor">MyThingVendor</property>
+             <property name="modelId">...</property>
+        </properties>
+		...
+    </thing-type>
+```
+
+The properties specified inside the thing type description XML file can be retrieved by the operation `getProperties` of the corresponding `org.eclipse.smarthome.core.thing.type.ThingType` instance. If a thing will be created for this thing type then its properties will be automatically copied into the new thing instance. Therefore the `org.eclipse.smarthome.core.thing.Thing` interface provides also the `getProperties` operation to retrieve the defined properties. Additional thing properties that are only known at runtime have to be set by the concrete thing handler. For this purpose the operation `setProperty` of the thing interface has to be used:
```

Comment:
This is the architecture page. So I would not refer to any Java code here, this should rather happen in the binding how-to.